### DISCUSSION TITLE
Trying to fine-tune Tacotron2-DDC checkpoint but coarse_decoder layers are missing

### DISCUSSION THREAD

**Discussion by serg06** [coqui-ai/TTS#584] [General Q&A] [answered]
Hello,

I want to finetune the `tts_models/en/ljspeech/tacotron2-DDC` model, but the checkpoint is missing `coarse_decoder` layers. Is there a way to get the full checkpoint that has those layers? @erogol I notice [you mentioned that such a checkpoint exists](http://disq.us/p/2bw3y8f)?

**Comment by erogol** [comment, marked as answer]
Unfortunately, I don't keep the full checkpoint.  

But you can initiate training from that model and start with a newly initialized coarse decoder. That'd not make a big difference.
